packages/google_identity_services_web/pubspec.yaml

```diff
@@ -2,14 +2,14 @@ name: google_identity_services_web
 description: A Dart JS-interop layer for Google Identity Services. Google's new sign-in SDK for Web that supports multiple types of credentials.
 repository: https://github.com/flutter/packages/tree/main/packages/google_identity_services_web
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+google_identiy_services_web%22
-version: 0.3.1+1
```

Comment:
Optional nit: This seems like a pretty trivial change; i would version it as `+2` instead.

Reply:
Rolled to +2!